src/test/java/com/google/cloud/tools/maven/it/AbstractSingleYamlDeployIntegrationTest.java

```diff
@@ -33,6 +33,7 @@ public void testDeployStandard()
     Verifier verifier = new StandardVerifier("testDeployStandard");
 
     // execute with staging directory not present
+    verifier.executeGoal("package");
```

Comment:
Should these remove the call to the goal individually?

Reply:
What do you mean?

Reply:
can you execute a list of goals?